grib/src/main/java/ucar/nc2/grib/grib2/table/Grib2Tables.java

```diff
@@ -38,6 +38,7 @@
  * @since 4/3/11
  */
 @Immutable
+// TODO version 6: remove implements TimeUnitConverter
```

Comment:
Changing this inheritance now would I think technically be a breaking change that effects ~10 other classes that inherit from this one. If we decide in version 6 that this class isn't part of the public API then it is easy to fix.